config/initializers/populate_sysadm_h2.rb

```diff
@@ -352,6 +352,20 @@ class PopulateSysadmH2 < ActiveRecord::Base
       );
       INSERT INTO SYSADM.PS_UCC_NOTENRL_RSN (EMPLID, ACAD_CAREER, INSTITUTION, STRM, CLASS_NBR, MESSAGE_NBR, ERROR_MESSAGE_TXT, UC_ENRL_LASTATTMPT, UC_REASON_DESC) VALUES('11667051','UGRD','UCB01','2178',12392,'213','Available seats are reserved.', PARSEDATETIME('2019-07-31', 'yyyy-mm-dd'),'Reserved Seats');
 
+      DROP TABLE IF EXISTS SYSADM.PS_UCC_CAND_TRMSTA;
+      CREATE TABLE SYSADM.PS_UCC_CAND_TRMSTA (
+        EMPLID VARCHAR2(11),
+        ACAD_CAREER	VARCHAR2(4),
+        ACAD_PROG	VARCHAR2(5),
+        ACAD_PLAN	VARCHAR2(10),
+        ACAD_SUB_PLAN	VARCHAR2(10),
+        UC_CANDCY_END_TERM	VARCHAR2(5),
+        UC_CANDIDACY_STAT	VARCHAR2(1),
+        INSTITUTION	VARCHAR2(5)
+      );
+
+      INSERT INTO SYSADM.PS_UCC_CAND_TRMSTA (EMPLID, ACAD_CAREER, ACAD_PROG, ACAD_PLAN, ACAD_SUB_PLAN,UC_CANDCY_END_TERM, UC_CANDIDACY_STAT, INSTITUTION) VALUES('11667051','GRAD','GACAD','79249PHDG','79249SP09G','2202','G','UCB01');
```

Comment:
@redconfetti -  a minor nit... here is the order of the fields in Campus Solutions record object:
![Screen Shot 2019-10-21 at 10 33 08 AM](https://user-images.githubusercontent.com/12177541/67224364-42344a80-f3ee-11e9-90c9-f90ff61e6a38.png)

Reply:
@marcpechaitis - So you want me to order them in the order in the table?

Reply:
@marcpechaitis - Okay, the view you're looking at has a different name than the one I'm looking at. Both are similar, but this was the one I was told to use.

![image](https://user-images.githubusercontent.com/746759/67252830-06909380-f475-11e9-8356-b89579baeecf.png)

Reply:
@redconfetti yes I think you should re-order them.

Reply:
@marcpechaitis - Done